addons/core/CBA_Settings.sqf

```diff
@@ -0,0 +1,474 @@
+// client Settings
+[
+    "TFAR_default_radioVolume",
+    "SLIDER",
+    ELSTRING(settings,default_radioVolume),
+    "Task Force Arrowhead Radio",
+    [1, 9, 7, 0],
+    0
+] call CBA_Settings_fnc_init;
+[
+    "TFAR_volumeModifier_forceSpeech", 
+    "CHECKBOX", 
+    [ELSTRING(settings,directSpeechModifier), ELSTRING(settings,directSpeechModifier_desc)], 
```

Comment:
ELSTRING is correct? You had some typo in the last PR.

Reply:
`ELSTRING` inside of `*.sqf` `ECSTRING` inside of configs